.env.example

```
@@ -46,6 +47,8 @@ KAKAO_CLIENT_SECRET=your-kakao-client-secret
 
 # OAuth
 OAUTH_REDIRECT_URI=/
+OAUTH_CALLBACK_BASE_URL=api.dev.timo.io.kr # 로컬 테스트의 경우 localhost:8080
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The OAUTH_CALLBACK_BASE_URL key should go before the OAUTH_REDIRECT_URI key

(UnorderedKey)